src/utils/vcs.rs

```diff
@@ -139,6 +139,8 @@ impl VcsUrl {
             static ref HOST_WITH_PORT: Regex = Regex::new(r"(.*):\d+$").unwrap();
             static ref GCB_GIT_PATH_RE: Regex =
                 Regex::new(r"^p/.+/r/github_(.+?)_(.+?)(?:\.git)?$").unwrap();
+            static ref BITBUCKET_SERVER_PATH_RE: Regex =
+                Regex::new(r"projects/(.+)/repos/(.+)/browse").unwrap();
```

Comment:
Is `/browse` also part of the git URL / ssh URL?

Reply:
Yes, it's part of the URL we provide - https://github.com/getsentry/sentry/blob/87a67caeebbadc1b2f40ab11fd99fe113feb7042/src/sentry/integrations/bitbucket_server/repository.py#L72-L75